Source/DafnyCore/AST/SyntaxDeserializer/HandWritten.cs

```diff
@@ -210,6 +210,14 @@ public T DeserializeGeneric<T>(System.Type actualType) {
       return (T)(object)ReadToken();
     }
 
+    if (actualType == typeof(MultiSelectExpr)) {
```

Comment:
Are these newly added ifs not already generated and added to the ReadObject method called on line 221?